Blackfynn/data-portal
# Add favicon #11xh61

# Description

The purpose of this PR is to add the favicon to the app.

## Type of change


# How Has This Been Tested?

- Open the app
- Click on all links in the main navigation
- Verify that the favicon is on all pages

# Checklist:
